x/fswap/spec/01_concepts.md

```diff
@@ -29,10 +43,11 @@ type Config struct {
 }
 ```
 
-## Proposal
+## MsgSetSwap
 
-Typically, a `Swap` is proposed and submitted through foundation via a `MsgSetSwap`.
-This proposal prescribes to the standard foundation process. If the proposal passes, the `Swap` can be used on chain.
+Typically, a `Swap` is proposed and submitted through `x/foundation` via a `MsgSetSwap`.
+Other modules (`x/gov`, `x/foundation`) will include this message in their proposals. If the proposal passes, the `Swap` can be used on chain.
```

Comment:
We should not mention the exact module name `x/gov` now, because it's not being supported yet.